packages/vue/src/components/molecules/SfSelect/_internal/SfSelectOption.vue

```diff
@@ -1,18 +1,23 @@
 <template>
   <li
+    v-focus
     class="sf-select-option"
     :class="{ 'sf-select-option--is-active': selected === value }"
     role="option"
     :aria-selected="selected === value ? 'true' : 'false'"
     @click="clicked"
+    @keyup.space.enter="clicked"
+    tabindex="0"
```

Comment:
@justyna-13 are you sure it's a good idea? when I testing https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select then I use arrow to set value ;)

Reply:
@pspaczek right :) we can use only arrows